cli/cmd/root_test.go

```diff
@@ -194,6 +197,49 @@ func Test_runVersionCheck(t *testing.T) {
 			metadataStatus: http.StatusInternalServerError,
 			wantOutput:     "* Warning: could not check Keptn server version: received invalid response from Keptn API\n",
 		},
+		{
+			name:            "skip version check for 'keptn install'",
```

Comment:
I am writing unit tests related `keptn install` in `root_test.go` because the code to skip version check on `install` subcommand is in `root.go`.